docs/source/package_reference/cards.mdx

```diff
@@ -0,0 +1,40 @@
+# Repository Cards
+
+The huggingface_hub library provides a Python interface to create, share, and update Model/Dataset Cards. 
+Visit the [dedicated documentation page](https://huggingface.co/docs/hub/models-cards) for a deeper view of what 
+Model Cards on the Hub are, and how they work under the hood. You can also check out our [Model Cards guide](../how-to-model-cards) to 
+get a feel for how you would use these utilities in your own projects.
+
+## Repo Card
+
+The `RepoCard` object is the parent class of [`ModelCard`] and [`DatasetCard`].
+
+[[autodoc]] huggingface_hub.repocard.RepoCard
+
```

Comment:
```suggestion
[[autodoc]] huggingface_hub.repocard.RepoCard
    - __init__
    - all
```

Repocard init does a lot of things and is really well documented so I would add it explicitely to the generated doc.